server/views/partials/header.njk

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
What does this bit do in the header?

Reply:
Thats just the SVG markup for the hmpps logo

Reply:
Yes, sorry I can see that now. Just looked a bit odd at first glance. All good.